composer.json

```diff
@@ -44,5 +44,11 @@
         "coding-standards-apply": [
             "@coding-standards-apply/phpcs"
         ]
+    },
+    "config": {
+        "allow-plugins": {
+            "cweagans/composer-patches": true,
```

Comment:
Are we gonna need patches here?

Reply:
No, but `os2forms/os2forms 3.5.0 requires cweagans/composer-patches` and running `composer install` in this repository will ask us to set this.